pages/agent/v3/cli_annotate.md

```diff
@@ -112,7 +112,7 @@ p4 pt4 pr4 pb4 pl4 py4 px4
 
 ### Colored console output
 
-Console output in annotations can be displayed with ANSI colors when wrapped in a Markdown block marked as `term` or `terminal` syntax. There is a limit of 10 term/terminal code blocks we'll render per annotation.
+Console output in annotations can be displayed with ANSI colors when wrapped in a Markdown block marked as `term` or `terminal` syntax. There is a limit of blocks per annotation.
```

Comment:
I made a mistake here, sorry! I didn't mean to remove the number for the limit 🤦🏻‍♂️

```suggestion
Console output in annotations can be displayed with ANSI colors when wrapped in a Markdown block marked as `term` or `terminal` syntax. There is a limit of 10 blocks per annotation.
```